docusaurus.config.js

```diff
@@ -152,13 +148,29 @@ const config = {
         },
         // ... other options
       },
+    ],[
+      '@docusaurus/plugin-content-docs',
+      {
+        id: 'specification',
+        path: 'external-content/specification/specification',
+        routeBasePath: 'specification',
+        editUrl: (params) => {
+          return `https://github.com/open-feature/spec/edit/main/${params.docPath}`;
+        },
+        // ... other options
+      },
     ],
     [
-      externalContentPlugin,
+      '@docusaurus/plugin-client-redirects',
       {
-        name: 'spec',
-        sourcePath: 'specification/specification',
-        destinationPath: 'specification',
+        createRedirects(existingPath) {
+          if (existingPath.includes('/docs/specification')) {
```

Comment:
we might need to change how the preview works - because the preview is still a development build, and the redirects only work in production.

If you like i can schedule a short meeting for the two of use, to verify this locally with a production build.

Reply:
i added netlify redirects, with those we should be safe for the future too :)